nx-X11/programs/Xserver/dix/property.c

```diff
@@ -531,6 +531,7 @@ ProcGetProperty(ClientPtr client)
 	pProp = pProp->next;
     }
 
+    memset(&reply, 0, sizeof(xGetPropertyReply));
```

Comment:
I have backported that single memset now, but not all the other stuff. dix/property.c does not have them either. So I consider this as done.